pkgs/development/python-modules/xformers/default.nix

```diff
@@ -80,6 +99,10 @@ buildPythonPackage {
     transformers
     timm
     # flash-attn
+    # Checks flash-attn version, but fails as no .git found.
+    # Should the check here allow for missing git?
+    # https://github.com/facebookresearch/xformers/blob/e6e66958b29be6ed6428ab0664092e1733d4bdbc/setup.py#L71
+    git
```

Comment:
you've added the patch to allow explicitely building without git.
So it should just show a git failure log but still succeed without this right ?

Reply:
Oh, yes, forgot about that, that's redundant.

Reply:
The nixpkgs review builds no problem. If you remove this, I'm happy to merge this and you can make an update to 0.24 whenever you have time (no rush).